LICENSE.md

```diff
@@ -3,7 +3,7 @@ If you do not agree to these terms, do not access Particular Software code.
 
```

Comment:
Suggestion for the first line:
By accessing code under the [Particular Software GitHub Organization](https://github.com/Particular) (Particular Software), you are hereby agreeing to the following licensing terms. If you do not agree to these terms, do not access Particular Software code.